### PR TITLE
 Add missing Entity Type options to the Import page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds missing Entity Type options to the Import page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/system/data-import.html

### CODE REVIEW

File: src/system/data-import.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Special characters (such as the equal sign, greater and less than symbols, singl
 
 When importing product data, new product data is added to existing product data entries in the database. All fields except SKU can be updated through import. All existing product data is replaced with the imported new data. Exercise caution when replacing data. All existing product data will be completely cleared and all references in the system will be lost.
 
-![Data import]({% link images/images/system-import.png %}){: .zoom}
```
Comment (dobooth) [inline src/system/data-import.md:19]: Please remove the old image if it is not used in another topic.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/system/data-import.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/system/data-import.md:19]: @dobooth 
Done, thank you!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1153`.

**Pull request merged**